src/helperFunctions/config.py

```diff
@@ -29,6 +28,14 @@ def get_config_dir():
 
 
 def read_list_from_config(config_file: ConfigParser, section: str, key: str, default=None):
+    '''
+    Parses a comma separated list in section `section` with key `key`.
+
+    :param section: The section to read from
```

Comment:
The first parameter is missing
```suggestion
    :param config_file: The FACT configuration
    :param section: The section to read from
```